src/spread/mult-obj-ident.case

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Object Spread operator following other properties
+template: default
+esid: pending
+includes: [propertyHelper.js]
+info: |
+    Pending Runtime Semantics: PropertyDefinitionEvaluation
+
+    PropertyDefinition:...AssignmentExpression
+
+    1. Let exprValue be the result of evaluating AssignmentExpression.
+    2. Let fromValue be GetValue(exprValue).
+    3. ReturnIfAbrupt(fromValue).
+    4. Let excludedNames be a new empty List.
+    5. Return CopyDataProperties(object, fromValue, excludedNames).
+
+---*/
+
+//- setup
+let o = {c: 3, d: 4};
+
+//- args
+{a: 1, b: 2, ...o}
+//- params
+obj
+//- body
+assert.sameValue(obj.a, 1);
+assert.sameValue(obj.b, 2);
+assert.sameValue(obj.c, 3);
+assert.sameValue(obj.d, 4);
+assert.sameValue(Object.keys(obj).length, 4);
+
+verifyEnumerable(obj, "c");
```

Comment:
I know we want to assert the new properties, but just to confirm, we should also check the `a` and `b` properties.